CHANGELOG.md

```diff
@@ -6,6 +6,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [15.0.0]
+### Changed
+- **BREAKING** Update TokensController allTokens state structure to make network/chainID parent of account ([#572](https://github.com/MetaMask/controllers/pull/572))
+  - The shape of allTokens state field on TokensController has been reorganized. Consumers of the TokensController will have to migrate existing token state to new shape.
+- **BREAKING** ignoredTokens changed to allIgnoredTokens ([#570](https://github.com/MetaMask/controllers/pull/570))
+  - a new state field on the TokensController - allIgnoredTokens - now manages ignoredTokens by network and accountAddress, ignoredTokens is now the array of token address strings (previously an array of full token objects) that have been hidden by the user for the currently active network and accountAddress pair. Consumers of the TokensController will have to migrate existing ignoredTokens array to allIgnoredTokens object.
+- **BREAKING** Improve BaseControllerV2 messenger type ([#556](https://github.com/MetaMask/controllers/pull/556))
+  - This is a breaking change, because anyone extending BaseControllerV2 will now be required to supply an additional generic parameter.
+- **BREAKING** Remove redundant default export from util.ts ([#574](https://github.com/MetaMask/controllers/pull/574))  
+  - This is potentially breaking for consumers who import methods from the utils.ts as default/unnamed imports and will require changing these to named imports.
```

Comment:
Nit: This was fine but the wording seemed a little confusing to me. here's a suggested improvement:
```suggestion
  - This is breaking for consumers who use the default import of the utils module, and will require using named imports instead.
```